pkg/controller/controller_broker_test.go

```diff
@@ -156,6 +156,9 @@ func TestShouldReconcileBroker(t *testing.T) {
 }
 
 func TestReconcileBrokerExistingServiceClass(t *testing.T) {
+	// Verify the fake clients perform as expected during reconcileBroker().
```

Comment:
I was looking for more of a qualitative description of what scenario this test is exercising